docs/contribute/03_style.qmd

```diff
@@ -16,22 +21,98 @@ pip install pre-commit
 pre-commit run --all-files
 ```
 ::: {.callout-note}
-Some of the packages needed to run the `pre-commit` linting can not be installed automatically (e.g. `prettier`, `actionlint`, `shellcheck`), and they need to be installed through a system package manager.
+Some of the packages needed to run the `pre-commit` linting can not be installed
+automatically (e.g. `prettier`, `actionlint`, `shellcheck`), and they need to be
+installed through a system package manager.
 :::
 
-Optionally, you may want to setup the `pre-commit` hooks to run automatically when making a git commit. To do this, run the following from the root of the Ibis repository:
+Optionally, you may want to setup the `pre-commit` hooks to run automatically
+when making a git commit. To do this, run the following from the root of the
+Ibis repository:
 
 ```sh
 pre-commit install
 ```
 
-This will run the code linters automatically when you make a git commit. If you want to skip these checks, do `git commit --no-verify`
+This will run the code linters automatically when you make a git commit. If you
+want to skip these checks, do `git commit --no-verify`
 
 
 ::: {.callout-tip}
-If you use `nix-shell`, all of these are already setup for you and ready to use, and you don't need to do anything to install these tools.
+If you use `nix-shell`, all of these are already setup for you and ready to use,
+and you don't need to do anything to install these tools.
 :::
 
 ## Docstrings
 We use [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) as our
 standard format for docstrings.
+
+## Documentation, blog, and other prose style
+
+Always capitalize Ibis in prose.
+
+### General points on style and word usage
+
+Avoid the passive voice.  Either Ibis does something or the user does
+something or a particular backend does something.
+
+Try to avoid using words like "simply", "simple", "obviously", "just", when
+describing things that Ibis makes easier.  Either the simplicity is
+self-evident, or it isn't and you are inadvertently insulting the reader.
+
+Prose should be authored by you™. We discourage LLM-written docs and posts, but
+you are free to use these tools to aid in your writing.
+
+### Text formatting
+
+Wrap long prose strings between 80-90 characters per line. It is very helpful
```

Comment:
Is it possible to just pick a number? :) Like 80 or 88?

Reply:
I can specify an upper bound, how about that?